posthog/hogql_queries/error_tracking_query_runner.py

```diff
@@ -134,22 +67,13 @@ def where(self):
             ast.Placeholder(expr=ast.Field(chain=["filters"])),
         ]
 
-        groups = []
-
-        if self.query.fingerprint:
-            groups.append(self.group_or_default(self.query.fingerprint))
-        elif self.query.assignee:
-            groups.extend(self.error_tracking_groups.values())
-
-        if groups:
+        if self.query.issueId:
             exprs.append(
-                ast.Call(
-                    name="has",
-                    args=[
-                        self.group_fingerprints(groups),
-                        self.extracted_fingerprint_property(),
-                    ],
-                ),
+                ast.CompareOperation(
+                    op=ast.CompareOperationOp.Eq,
+                    left=ast.Field(chain=["properties", "$exception_issue_id"]),
+                    right=ast.Constant(value=self.query.issueId),
+                )
             )
 
         if self.query.searchQuery:
```

Comment:
We should handle the TODO's in here at some point I guess, we're a ways of it yet though